tests/test.js

```diff
@@ -1,4 +1,4 @@
-import preslovi from '@pionir/preslovljivac-js';
+import preslovi from '../main';
```

Comment:
Tests in the package should point to relative paths/files instead of npm library resolving.